qemu/tests/cfg/serial_modprobe.cfg

```diff
@@ -0,0 +1,14 @@
+- serial_modprobe: install setup image_copy unattended_install.cdrom
+    type = virt_subtest_combine
+    kill_vm_on_error = yes
+    virtio_ports_fixed = "vs1"
+    vrtio_port_type_fixed = "serialport"
+    only Linux
+    login_timeout = 240
+    subtests = check_device remove_mod check_no_mod check_no_device load_mod check_device
+    cmd_check_device = ls /dev/vport0p1
```

Comment:
do we need to check the device in the output,
if yes, didn't find the step in the code
if no, what's the step used to,
suggest to print the output as a reference for debugging
and print out the cmd should be more clear for testing in my opinion, we can fix in another patch.

Thanks